### ISSUE TITLE
Problem with running on Windows

### ISSUE THREAD

**Issue by OmarAshkar** [RichardHooijmaijers/shinyMixR#17]
Hello,

I am facing a problem using ShinyMixR on windows, running on linux is fine. I am getting the following error when running the example run1:

```
Listening on http://127.0.0.1:3011
Browsing http://127.0.0.1:3011
Warning in gzfile(file, "rb") :
  cannot open compressed file 'shinyMixR/run1.res.rds', probable reason 'No such file or directory'
Error in gzfile(file, "rb") : cannot open the connection
Warning in gzfile(file, "rb") :
  cannot open compressed file 'shinyMixR/run1.res.rds', probable reason 'No such file or directory'
Error in gzfile(file, "rb") : cannot open the connection
Warning: Error in ||: 'length = 2' in coercion to 'logical(1)'
  2: shiny::runApp
  1: shinyMixR::run_shinymixr
```
and the app will fail completely.

```
sessionInfo()
R version 4.3.0 (2023-04-21 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)  
Running under: Windows 10 x64 (build 19044)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252 LC_NUMERIC=C                           LC_TIME=English_United States.1252

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] shinyMixR_0.4.0 ggplot2_3.4.2   shiny_1.7.4

loaded via a namespace (and not attached):
 [1] vctrs_0.6.2      cli_3.6.1        rlang_1.1.1      generics_0.1.3   promises_1.2.0.1 jsonlite_1.8.4   xtable_1.8-4     glue_1.6.2       colorspace_2.1-0 htmltools_0.5.5  httpuv_1.6.11    scales_1.2.1
[13] fansi_1.0.4      grid_4.3.0       munsell_0.5.0    tibble_3.2.1     ellipsis_0.3.2   fastmap_1.1.1    lifecycle_1.0.3  compiler_4.3.0   dplyr_1.1.2      Rcpp_1.0.10      pkgconfig_2.0.3  later_1.3.1     
[25] digest_0.6.31    R6_2.5.1         tidyselect_1.2.0 utf8_1.2.3       pillar_1.9.0     magrittr_2.0.3   withr_2.5.0      gtable_0.3.3     mime_0.12
```

Thanks!

**Comment by RichardHooijmaijers** [comment]
Just for my understanding, did you work on the project folder in windows that you started in the linux environment?
Currently the app saves full path names at times (will change in the future). This can cause that the folder structure cannot be directly used when copied or initiated in a different OS. There is a quick solution for now; delete the file "project.rds" in the shinyMixR subfolder. This file will be automatically created when the app starts so no work will be lost. Not sure if this helps in your case but could you please try this?

**Comment by RichardHooijmaijers** [comment]
Hi @OmarAshkar, Took some digging because I couldn't reproduce this on my system. Then I noticed you are running R4.3.0. Within this version if statements resulting in vectors > 1 will give an error. I found this main issue was actually caused by a function I use from another package. I double checked my code and made some other improvements. Hopefully the issue is gone when you install the latest github version of `shinyMixR`. Let me know if it works for you or  have any other issues
